sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -660,12 +660,21 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+
+  val QUERY_EXECUTION_LISTENERS =
+    ConfigBuilder("spark.sql.queryExecutionListeners")
+      .doc("QueryExecutionListeners to be attached to the SparkSession")
```

Comment:
We do not have a separate document for the Spark SQL configuration. We expect users to do it using the command `set -v`. This command will output the contents of `doc`.